src/protocols.json

```diff
@@ -47,7 +47,7 @@
       "url": "https://berachain.com"
     },
     {
-      "description": "Native liquidity marketplace on Berachain.",
+      "description": "Beradrome is Berachain’s native restaking & liquidity marketplace",
```

Comment:
The name should not be in the description. Please ignore the design copy and revert.

Reply:
thats not design copy, thats from their website.